### PR TITLE
PBW cleanup

### PR DESCRIPTION
**Pull request by mistermendez**
Main updates:

1.) reorder propTypes and defaultProps to JSX style guide
2.) Use Responsive Manager componentWidth instead of controlBarWidth
3.) Use Responsive Manager to detect resize
4.) remove unused code

### CODE REVIEW

File: js/components/scrubberBar.js [hunk 1]
```diff
@@ -10,32 +10,13 @@ var React = require('react'),
...
-          }
-        }
-      }
-    };
-  },
-
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
-    this.responsiveUIMultiple = this.getResponsiveUIMultiple(this.props.responsiveView);
-    this.lastScrubX = null;
```
Comment (chibzu) [inline js/components/scrubberBar.js]: lastScrubX is just bookkeeping, does it need to be in the React state? I don't think there's a reason to trigger a redraw when it changes.

File: js/components/scrubberBar.js [hunk 2]
```diff
@@ -126,13 +98,13 @@ var ScrubberBar = React.createClass({
...
-      var scrubbingPlayheadX = this.props.currentPlayhead * this.scrubberBarWidth / this.props.duration + deltaX;
-      this.props.controller.updateSeekingPlayhead((scrubbingPlayheadX / this.scrubberBarWidth) * this.props.duration);
+      var deltaX = evt.clientX - this.state.lastScrubX;
+      var scrubbingPlayheadX = this.props.currentPlayhead * this.state.scrubberBarWidth / this.props.duration + deltaX;
+      this.props.controller.updateSeekingPlayhead((scrubbingPlayheadX / this.state.scrubberBarWidth) * this.props.duration);
       this.setState({
         scrubbingPlayheadX: scrubbingPlayheadX
       });
-      this.lastScrubX = evt.clientX;
+      this.state.lastScrubX = evt.clientX;
```
Comment (chibzu) [inline js/components/scrubberBar.js]: From React's docs:

> NEVER mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.

File: js/components/scrubberBar.js [hunk 3]
```diff
@@ -146,7 +118,7 @@ var ScrubberBar = React.createClass({
     evt.stopPropagation(); // W3C
     evt.cancelBubble = true; // IE
 
-    this.lastScrubX = null;
+    this.state.lastScrubX = null;
```
Comment (chibzu) [inline js/components/scrubberBar.js]: Ditto here.

**Comment by mistermendez** [comment]
@chibzu moved lastScrubX back to instance var

**Comment by chibzu** [comment]
Wunderbar

**Pull request merged**